recipes/c-progressbar/bld.bat

```diff
@@ -0,0 +1,15 @@
+mkdir build
+if errorlevel 1 exit 1
+cd build
+if errorlevel 1 exit 1
+cmake -G "NMake Makefiles" ^
+         -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE ^
+         -DCMAKE_INSTALL_PREFIX:PATH=%PREFIX% ^
+         ..
+if errorlevel 1 exit 1
+nmake
+if errorlevel 1 exit 1
+nmake test
+if errorlevel 1 exit 1
+nmake install
+if errorlevel 1 exit 1
```

Comment:
Please drop this file if we are not building on Windows. Can always readd all of this with a PR to the feedstock.